josueoaraujo/slack-jira-app
# Zen 655 remove old slack zenbot

First app version

Testing:

 Using:
- The .env I sent by Slack
- The Slack Worspace: Support Slack Test

Steps:

- Invite the bot (Test Bot) to the desired channel 
- Send a message containing 1 or 2 jira issues: the bot should return the information right away.
- Send a message containing >=3 jira issues: the bot should ask if the user want to receive the information.
